pyndustric/compiler.py

```diff
@@ -702,6 +702,26 @@ def emit_unit_syscall(self, node: ast.Call):
             else:
                 raise CompilerError(ERR_BAD_SYSCALL_ARGS, node)
 
+        elif method == "target":
+            if len(node.args) == 2:
+                x, y = map(self.as_value, node.args)
+                self.ins_append(f"ucontrol target {x} {y} 0 0 0")
+            elif len(node.args) == 3:
+                x, y, shoot = map(self.as_value, node.args)
+                self.ins_append(f"ucontrol target {x} {y} {shoot} 0 0")
+            else:
+                raise CompilerError(ERR_BAD_SYSCALL_ARGS, node)
+
+        elif method == "target_prediction":
```

Comment:
If the other change is made, this will need to be renamed too:

```suggestion
        elif method == "target_unit":
```

Reply:
My vision for this project is a Python compiler to assembly and I think we should stick with more Pythonic names, while improving naming wherever it's possible as long as it makes sense. I definitely wouldn't think at first glance "target prediction requires a unit" but target unit seems to make more sense.